metals/src/main/scala/scala/meta/internal/metals/doctor/DoctorResults.scala

```diff
@@ -152,3 +154,33 @@ final case class DoctorFolderHeader(
     base
   }
 }
+
+/**
+ * Information about an error report.
+ * @param name display name of the error
+ * @param timestamp date and time timestamp of the report
+ * @param uri error report file uri
+ * @param buildTarget optional build target that error is associated with
+ * @param shortSummary short error summary
+ * @param errorReportType one of "metals", "metals-full", "bloop"
+ */
+final case class ErrorReportInfo(
+    name: String,
+    timestamp: Long,
```

Comment:
I'm sending a timestamp but I wonder if it'd be better to send nicely formatted time-date string instead.